x/oracle/keeper/keeper.go

```diff
@@ -86,24 +86,23 @@ func (k Keeper) ProcessClaim(ctx sdk.Context, claim types.Claim) (types.Status,
 	}
 
 	if !inWhiteList {
-		fmt.Println("sifnode oracle keeper ProcessClaim validator no in whitelist")
+		sugaredLogger.Errorw("sifnode oracle keeper ProcessClaim validator no in whitelist.")
 		return types.Status{}, types.ErrValidatorNotInWhiteList
 	}
 
 	activeValidator := k.checkActiveValidator(ctx, claim.ValidatorAddress)
 	if !activeValidator {
-		fmt.Println("sifnode oracle keeper ProcessClaim validator not active")
+		sugaredLogger.Errorw("sifnode oracle keeper ProcessClaim validator no active.")
```

Comment:
error message should be:
"sifnode oracle keeper ProcessClaim validator not active"